docker/Dockerfile

```diff
@@ -9,11 +9,14 @@ COPY . /polkadot
 
 RUN apt-get update && \
 	apt-get upgrade -y && \
-	apt-get install -y cmake pkg-config libssl-dev git
-
+	apt-get install -y cmake pkg-config libssl-dev git clang
 RUN curl https://sh.rustup.rs -sSf | sh -s -- -y && \
-	export PATH=$PATH:$HOME/.cargo/bin && \
-	cargo build --$PROFILE
+        export PATH=$PATH:$HOME/.cargo/bin && \
+        rustup toolchain install nightly && \
+        rustup target add wasm32-unknown-unknown --toolchain nightly && \
+        cargo install --git https://github.com/alexcrichton/wasm-gc && \
+        scripts/build.sh && \
```

Comment:
```suggestion
```

master doesn't use `build.sh` anymore

Reply:
should  I remove `scripts/build.sh` from git tree?

Reply:
There's no need to remove it in this branch since it was already removed in master.